catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -498,6 +501,14 @@ public void setIngest(Boolean isIngest) {
         this.isIngest = isIngest;
     }
 
+    public Boolean getAllowEditing() {
```

Comment:
Maybe change the variable to `editingAllowed` and this method to `isEditingAllowed`